### PR TITLE
Update deployment-guide.mdx

### PR DESCRIPTION
**Pull request by gtelnet**
updated formatting/spelling; added info on heartbeats

### CODE REVIEW

File: docs/mine-hnt/validators/mainnet/deployment-guide.mdx [hunk 1]
```diff
@@ -131,11 +131,11 @@ package by PaulVMo, is the least complex to deploy and has very high performance
 has the highest level of complexity to deploy, requires a higher level of Linux technical expertise,
 and also has very high performance. The docker version is less complex to deploy than Source, if you
 already have experience with Docker, but does not perform as well as the other two options. The
-debian package and docker options are also the simplest to deploy and maintain when running multiple
-validator instances on a single host.
+Debian package is also the simplest to deploy and maintain when running multiple validator instances on a single host.
```
Comment (abhay) [inline docs/mine-hnt/validators/mainnet/deployment-guide.mdx]: Is this true? I'd expect Docker to be the way to manage multiple on a single host.

**Review by abhay — DISMISSED** [review]
Generally LGTM. one question and would appreciate some of the longer lines to be wrapped. We should have some automation in the repo to help with this.

File: docs/mine-hnt/validators/mainnet/deployment-guide.mdx (hunk 1 above)
Comment (gtelnet) [reply docs/mine-hnt/validators/mainnet/deployment-guide.mdx]: Yes, based on how Paul configured the deb packages, it automatically increments the port mappings for both 215x and 808x for each instance, which removes all the confusion around configuring port mappings for each docker instance, which often leads to support questions in discord for people who aren't experts in docker.

**Comment by gtelnet** [comment]
> 

What's the process for wrapping the lines? Thanks!

**Comment by samgutentag** [comment]
hey @gtelnet best way to wrap lines and tidy us is to run `npx prettier --write --prose-wrap always <target>`

so in your case:
```bash
npx prettier --write --prose-wrap always docs/mine-hnt/validators/mainnet/deployment-guide.mdx
```

if you are using VSCode the master branch has an included `.prettierrc` file that will format on save if you enable that as well. Working on getting this added to the repo automation soon

**Comment by samgutentag** [comment]
went ahead and applied line wraps and formatting fixes 😁

**Review by samgutentag — APPROVED** [review]

**Review by samgutentag — APPROVED** [review]

**Comment by samgutentag** [comment]
@abhay it looks like you need to sign off on your request and then we are good to merge

**Comment by gtelnet** [comment]
bump

**Review by samgutentag — APPROVED** [review]
got lost in the shuffle, sorry for the delay

**Pull request merged**